content/the-ember-times-issue-175.md

```diff
@@ -11,22 +12,26 @@ tags:
 ---
 
 👋 Emberistas! 🐹
-
+Read the RFC on official TypeScript support on Ember ✅,
 `ember-responsive-image` v2 released 🖼️,
 Use Mirage JS to test non-trivial user flows ⬆️,
 ...
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [1. Section title in sentence case 🐹](section-url)
+## [RFC: Official TypeScript support ✅](https://github.com/emberjs/rfcs/pull/724)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) put up an [RFC for Official TypeScript support](https://github.com/emberjs/rfcs/pull/724) in Ember. The goal of the RFC is to determine if we should support [TypeScript](https://www.typescriptlang.org/) as a first-class citizen in Ember, and if yes, identify any gaps present today.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+> Ember officially supporting TypeScript means: All libraries which are installed as part of the default blueprint must ship accurate and up-to-date type definitions for the current edition. These types will uphold a Semantic Versioning commitment which includes a definition of SemVer for TypeScript types as well as a specification of supported compiler versions and settings, so that TypeScript will receive the same stability commitments as the rest of Ember.
+
+- Any addon that is part of the default blueprint must ship types.
+- The RFC explicitly does not require support for “typed templates”
+- Instead of opting into TypeScript using [ember-cli-typescript](https://github.com/typed-ember/ember-cli-typescript) it should be done by default
```

Comment:
Note: this is actually an open question! We should *support* it by default, but it's quite possible that we'll still be using (an updated version of) ember-cli-typescript for it just like we support Babel "out of the box" but do so via ember-cli-babel. (My personal opinion is that that's *likely*, in fact, but we'll need to hash it out in a future RFC!)

Reply:
@chriskrycho Hi Chris, I'm working on tying up loose ends for publish tonite. If you have time, can you please submit a GitHub suggestion / PR for the changes you want? No worries if not, I will take my best stab at it later on.

Reply:
I'll submit a PR in a couple minutes!

Reply:
See #933!